src/modules/companies/companies.service.ts

```diff
@@ -70,13 +71,33 @@ export class CompaniesService {
       const company = await this.findOne(id);
 
       if (!company)
-        throw new NotFoundException(`Company with ID ${id} not found`);
-
+        throw new NotFoundException(`Company with ID ${id} is not found`);
+
+      if (
+        updateCompaniesDto.organization_name &&
+        (await this.companyRepository.findOne({
```

Comment:
I dont like query in "if", create a constant above which will have query results and use this constant in "if"